docs/python/linting.md

```diff
@@ -8,278 +8,75 @@ DateApproved: 4/30/2022
+| Linter | Extension                                                                       |
+| ------ | ------------------------------------------------------------------------------- |
+| Pylint | https://marketplace.visualstudio.com/items?itemName=ms-python.pylint            |
+| flake8 | https://marketplace.visualstudio.com/items?itemName=ms-python.flake8            |
+| mypy   | https://marketplace.visualstudio.com/items?itemName=ms-python.mypy-type-checker |
 
-![Prompt for installing a linter](images/linting/install-linter-message.png)
+Linting extensions offered by the community:
 
-> **Note**: If you're using a global environment and VS Code is not running elevated, linter installation may fail. In that case, either run VS Code elevated, or manually run the Python package manager to install the linter at an elevated command prompt for the same environment: for example `sudo pip3 install pylint` (macOS/Linux) or `pip install pylint` (Windows, at an elevated prompt).
+| Linter | Extension                                                              |
+| ------ | ---------------------------------------------------------------------- |
+| Ruff   | https://marketplace.visualstudio.com/items?itemName=charliermarsh.ruff |
+| mypy   | https://marketplace.visualstudio.com/items?itemName=matangover.mypy    |
 
-## Disable linting
+> **Note**: If you don't find your preferred linter in the table above or in the marketplace, you can add support via an extension. The [Python Extension Template](/api/advanced-topics/python-extension-template.md) makes it easy to integrate new Python tools into VS Code.
 
-You can easily toggle between enabling and disabling your linter. To switch, open the Command Palette (`kb(workbench.action.showCommands)`) and select the **Python: Enable/Disable Linting** command.
+## General Settings
 
-This will populate a dropdown with the current linting state and options to **Enable** or **Disable** Python linting.
+| Setting        | Default      | Description                                                                                                                                                                                            |
+| -------------- | ------------ | ------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------ |
+| args           | `[]`         | Will be passed to the linter. **Note**: The officially supported linters run on individual opened file. Make sure your configuration applies in that scenario.                                         |
+| importStrategy | `useBundled` | `useBundled` means extension uses bundled version of the linter. `fromEnvironment` means it will attempt to load from your selected python environment first, otherwise falls back to bundled version. |
```

Comment:
```suggestion
| importStrategy | `useBundled` | When set to `useBundled`, the extension will use the version of the tool that it ships with. When set to `fromEnvironment`, it will attempt to load from your selected Python environment first, otherwise it will fall back to the bundled version. |
```